airflow-core/src/airflow/config_templates/config.yml

```
@@ -2663,6 +2663,22 @@ dag_processor:
       type: boolean
       example: ~
       default: "True"
+    dag_stability_check_level:
+      description: |
+        Controls the behavior of dag stability checker performed before DAG parsing in the dag-processor.
+        Static checks detect potential issues such as runtime-varying values in DAG/Task constructors
+        that could cause DAG version inflation.
```

```suggestion
        Controls the behavior of dag stability checker performed before Dag parsing in the dag-processor.
        The check detects potential issues such as runtime-varying values in Dag/Task constructors
        that could cause Dag version inflation.
```
We might need to word the first line to reflect exactly that this is about version inflation